src/main/java/net/minecraftforge/client/model/generators/BlockModelProvider.java

```diff
@@ -15,8 +16,16 @@
  */
 public abstract class BlockModelProvider extends ModelProvider<BlockModelBuilder> {
 
+    public BlockModelProvider(PackOutput output, String modid, ExistingFileHelper existingFileHelper) {
+        super(output, modid, BLOCK_FOLDER, BlockModelBuilder::new, existingFileHelper);
+    }
+
+    /**
+     * @deprecated Use {@link #BlockModelProvider(PackOutput, String, ExistingFileHelper)} instead
+     */
+    @Deprecated(forRemoval = true, since = "1.19.3")
```

Comment:
Should we be deprecating for removal, or removing entirely? I'm thinking because we're in the breaking changes period, we should be removing them now.

Reply:
Just for the record, considering this is datagen. Which the majority of users will never run as it's the job of the mod author to run it. I am far more lenient with breaking changes. Combined with the fact that we are in the breaking changes window of what we are treating as a new major version.

Reply:
Removal shall it be.